app/src/test/java/comp3350/mealbuddy/tests/objects/AccountTest.java

```diff
@@ -0,0 +1,23 @@
+package comp3350.mealbuddy.tests.objects;
+import org.junit.Assert;
+import org.junit.Test;
+import comp3350.mealbuddy.objects.Account;
+import comp3350.mealbuddy.objects.UserInfo;
+
+
+public class AccountTest {
+
+
+    public void test_Constructor() {
```

Comment:
I don't think we need to test constructors.  Pretty sure Braico said this as well. Not in this trivial way anyways